### ISSUE TITLE
Should not trigger when format expression contains backslash

### ISSUE THREAD

**Issue by nyuszika7h** [MichaelKim0407/flake8-use-fstring#21]
The following code triggers an FS002 error:
```py
"x={}".format(s.replace("\\", "/"))
```

However, attempting to change it to an f-string results in a syntax error:
```py
f"x={s.replace('\\', '/')}"
```
```pytb
SyntaxError: f-string expression part cannot include a backslash
```

**Comment by MichaelKim0407** [comment]
To be honest I think this issue is out of scope.

This plugin simply checks if `.format` is used. It does not analyze the syntax or the lexical structure. To correctly cover this kind of edge case would make the implementation way too complicated.

You can use `#noqa` to disable checking on this line. Or you can break the statement into two lines.
